### PR TITLE
feat: Improve geo data card

### PR DESCRIPTION
**Pull request by ptbrowne**
- Show times at the level of the arrows (we navigate in time)
- Show duration / distance and transport mode

### CODE REVIEW

File: packages/cozy-harvest-lib/src/datacards/GeoDataCard.jsx [hunk 1]
```diff
@@ -223,7 +274,7 @@ const makeQueryFromProps = ({ accountId }) => ({
     .where({
       'cozyMetadata.sourceAccount': accountId
     })
-    .sortBy([{ 'cozyMetadata.sourceAccount': 'desc' }, { startDate: 'desc' }])
+    .sortBy([{ 'cozyMetadata.sourceAccount': 'asc' }, { startDate: 'asc' }])
```
[the remark this answers is not in the document]
Comment (ptbrowne) [reply packages/cozy-harvest-lib/src/datacards/GeoDataCard.jsx:277]: No we render them in the ascending order in the carrousel, *but* we start at the last slide.

[the remark this answers is not in the document]
Comment (paultranvan) [reply packages/cozy-harvest-lib/src/datacards/GeoDataCard.jsx:271]: Ok but if you have, say, 100 trips ranging from january to march, we want to show the last 5 trips, right? With this query, we will show the first 5 trips from january.

File: packages/cozy-harvest-lib/src/locales/en.json [hunk 2]
```diff
@@ -489,7 +489,16 @@
...
-      "caption": "See your 5 most recent trips retrieved on %{konnectorName}"
+      "caption": "See your 5 most recent trips retrieved on %{konnectorName}",
+      "modes": {
+        "AIR_OR_HSR": "Plane",
+        "BICYCLING": "Bike",
+        "CAR": "Car",
+        "SUBWAY": "Subway",
+        "TRAIN": "Train",
+        "UNKNOWN": "",
+        "WALKING": "Walking"
```
Comment (paultranvan) [inline packages/cozy-harvest-lib/src/locales/en.json:500]: How did you find the available modes?

Comment (ptbrowne) [reply packages/cozy-harvest-lib/src/locales/en.json:500]: By looking at the data

**Review by paultranvan — APPROVED** [review]
:tada:

File: packages/cozy-harvest-lib/src/locales/en.json (hunk 2 above)
Comment (paultranvan) [reply packages/cozy-harvest-lib/src/locales/en.json:500]: Looks like the data was complete: https://github.com/e-mission/e-mission-server/blob/0475b456db197edacb524c5ab7a1124cd33b5333/emission/core/wrapper/modeprediction.py#L16

**Pull request merged**